lib/ui/chat/widgets/message_widget.dart

```diff
@@ -265,89 +273,89 @@ class ReactionsRow extends StatelessWidget {
-        Expanded(
-          child: Wrap(
-            spacing: 8.w,
-            children: [
-              ...(() {
-                final reactionGroups = <String, List<Reaction>>{};
-                for (final reaction in message.reactions) {
-                  reactionGroups.putIfAbsent(reaction.emoji, () => []).add(reaction);
-                }
-                return reactionGroups.entries.take(3).map((entry) {
-                  final emoji = entry.key;
-                  final count = entry.value.length;
-                  return GestureDetector(
-                    onTap: () {
-                      // Call the reaction tap handler to add/remove reaction
-                      onReactionTap?.call(emoji);
-                    },
-                    child: Container(
-                      padding: EdgeInsets.symmetric(horizontal: 8.w, vertical: 4.h),
-                      decoration: BoxDecoration(
-                        color:
-                            message.isMe
-                                ? context.colors.primary.withValues(alpha: 0.1)
-                                : context.colors.secondary.withValues(alpha: 0.1),
-                        borderRadius: BorderRadius.circular(12.r),
-                      ),
-                      child: RichText(
-                        text: TextSpan(
-                          children: [
-                            TextSpan(
-                              text: emoji,
-                              style: TextStyle(
-                                fontSize: 14.sp,
-                                color:
-                                    message.isMe
-                                        ? context.colors.primaryForeground
-                                        : context.colors.mutedForeground,
-                              ),
-                            ),
-                            TextSpan(
-                              text: ' ${count > 99 ? '99+' : count}',
-                              style: TextStyle(
-                                fontSize: 12.sp,
-                                fontWeight: FontWeight.w600,
-                                color:
-                                    message.isMe
-                                        ? context.colors.primaryForeground
-                                        : context.colors.mutedForeground,
-                              ),
-                            ),
-                          ],
+        ...(() {
+          final reactionGroups = <String, List<Reaction>>{};
+          for (final reaction in message.reactions) {
+            reactionGroups.putIfAbsent(reaction.emoji, () => []).add(reaction);
+          }
+          return reactionGroups.entries.take(3).map((entry) {
+            final emoji = entry.key;
+            final count = entry.value.length;
+            return GestureDetector(
+              onTap: () {
+                // Call the reaction tap handler to add/remove reaction
```

Comment:
[sug] I think the code already explains itself 😉  (which is good!)  So I'd remove this comment